front/app/components/Sharing/SharingButtons/index.tsx

```diff
@@ -46,6 +48,9 @@ const SharingButtons = ({
 }: Props) => {
   const [searchParams] = useSearchParams();
   const phaseContext = searchParams.get('phase_context');
+  const { data: appConfiguration } = useAppConfiguration();
+  const isSharingEnabled =
+    appConfiguration?.data.attributes.settings?.core.allow_sharing;
```

Comment:
Will all existing platforms have this set initially? If not then do we need to default it to true if `allow_sharing` is undefined?

Reply:
The typing would imply that it can be undefined